src/components/ui/ErrorMessage.tsx

```diff
@@ -0,0 +1,35 @@
+import { ErrorText } from 'components/ui/Text'
+import { useState } from 'preact/hooks'
+import { whitespace } from 'classnames/tailwind'
+import classNamesToString from 'helpers/classNamesToString'
+
+const truncateClass = (truncate?: boolean) =>
+  classNamesToString(
+    whitespace('whitespace-normal'),
+    truncate ? 'line-clamp-2' : undefined
+  )
+
+export default function ({
+  text,
+  truncated,
+}: {
+  text: string
+  truncated?: boolean
+}) {
+  const [isErrorOpen, setOpenError] = useState(false)
+
+  if (!truncated)
```

Comment:
Why do you check `if (!..)` if you can check `if (...)`? It's easier to read + your PC don't need to convert the thing

Reply:
Changed it